src/app/api/v1/user/remove-email/route.ts

```diff
@@ -50,7 +49,7 @@ export async function POST(req: NextRequest) {
         existingEmail.email,
       );
       return NextResponse.redirect(
-        AppConstants.SERVER_URL + "/user/settings",
+        process.env.SERVER_URL + "/user/settings",
```

Comment:
Same question re: `getEnvVarsOrThrow` vs. `process.env`

Reply:
Same reasoning as https://github.com/mozilla/blurts-server/pull/5044#discussion_r1756579517 - in this case, if it's unset, it would fall back to redirecting to `/user/settings` anyway, I think.